packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -74,32 +75,18 @@ const extractAttachedFilesMetadata = (metadata: Record<string, string>): FileMet
     return [];
   }
 };
-/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+/* @conditional-compile-remove(file-sharing) */
 const extractTeamsAttachmentsMetadata = (
-  attachments: ChatAttachment[],
-  resourceCache?: Record<string, string>
+  attachments: ChatAttachment[]
 ): {
   /* @conditional-compile-remove(file-sharing) */ files: FileMetadata[];
-  /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */ inlineImages: InlineImageMetadata[];
 } => {
   /* @conditional-compile-remove(file-sharing) */
   const files: FileMetadata[] = [];
-  /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
```

Comment:
These should only be removed after the component has shipped and released.

Reply:
Removing this instance is fine, since the remaining logic after the refactor is specific to file sharing.